atlascope/tests/test_dataloader.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+from django.contrib.auth.models import User
+import pytest
+from rest_framework import serializers
+
+from atlascope.core import models as atlascope_models
+from atlascope.core.management.commands.populate import DATALOADER_DIR, MODEL_JSON_MAPPING
+from atlascope.core.rest.additional_serializers import UserSerializer
+
+
+@pytest.mark.django_db
+def test_serializers_valid():
+    model_serializers = {
+        clas.Meta.model: clas
+        for clas in atlascope_models.__all__
+        if issubclass(clas, serializers.ModelSerializer)
+    }
+    model_serializers[User] = UserSerializer
+
+    for model, filename in MODEL_JSON_MAPPING:
+        objects = json.load(open(DATALOADER_DIR + filename))
+        for data_obj in objects:
+            assert model_serializers[model](data=data_obj).is_valid()
```

Comment:
```suggestion
            model_serializers[model](data=data_obj).is_valid(raise_exception=True)
```

This might provide more meaningful test failures.

Reply:
Changed in 7186274. It appears it's already giving us a detailed error; it's catching on the dataset whose importer is "vandy" and the tox environment does not have our custom plugins installed. Do you know how to include this in our tox env?

Reply:
> Do you know how to include this in our tox env?

I think so. I'll make a PR for that now.